schemas/bolt-config.schema.json

```diff
@@ -0,0 +1,200 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Bolt Config",
+  "description": "Bolt Config bolt.yaml Schema",
+  "type": "object",
+  "additionalProperties": false,
```

Comment:
we should only use this where bolt will blow up on unexpected keys.

Reply:
Updated. Looks like the only place Bolt will blow up with unrecognized keys is `inventory.yaml` for groups, targets, and config.